app/scripts/services/sql.js

```diff
@@ -100,14 +106,30 @@ angular.module('sql', [])
         canceler.reject();
       };
 
-      $http.post(baseURI("/_sql"), data, {'timeout': canceler.promise})
+      var request = {
+        url: baseURI("/_sql"),
+        method: "POST",
+        data: data,
+        config: {
+          'timeout': canceler.promise
+        }
+      };
+
+      if (errorTrace == true) {
```

Comment:
I would check for `errorTrace === true` and then set the `error_trace` param to `1` (or `true`).